client/src/featureform/resources.py

```diff
@@ -93,33 +93,7 @@ def serialize(self) -> bytes:
         return bytes(json.dumps(config), "utf-8")
 
 
-@typechecked
-@dataclass
-class RedshiftConfig:
```

Comment:
why remove this?

Reply:
my client version must have not been pulled with the latest redshift changes